src/orders/components/OrderInvoiceList/OrderInvoiceList.tsx

```diff
@@ -67,10 +71,12 @@ const OrderInvoiceList: React.FC<OrderInvoiceListProps> = props => {
           )
         }
       />
-      <CardContent className={invoices?.length && classes.cardContentTable}>
-        {!invoices ? (
+      <CardContent
+        className={generatedInvoices?.length && classes.cardContentTable}
+      >
+        {!generatedInvoices ? (
           <Skeleton />
-        ) : !invoices?.length ? (
+        ) : !generatedInvoices?.length ? (
```

Comment:
Please add stories covering `undefined`, `[]` and non-empty state

Reply:
added